docs/docs/node-api-helpers.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Node API Helpers
+jsdoc: ["gatsby/src/utils/api-node-helpers-docs.js"]
+apiCalls: NodeAPIHelpers
+contentsHeading: Shared helpers
+showTopLevelSignatures: true
+---
+
+The first argument passed to each of [Gatsby’s Node APIs](/docs/node-apis/) is an object containing a set of helpers. Helpers shared by all Gatsby’s Node APIs are documented in [Shared helpers](#shared-helpers) section.
+
+```javascript
+// in gatsby-node.js
+exports.createPages = gatsbyNodeHelpers => {
+  const { actions, reporter } = gatsbyNodeHelpers
+  // use helpers
+}
+```
+
+Common convention is to destructure helpers right in argument list:
+
+```javascript
+// in gatsby-node.js
+exports.createPages = ({ actions, reporter }) => {
+  // use helpers
+}
+```
+
+## Note
+
+Some APIs provide additional helpers. For example `createPages` provides `graphql` function. Check documentation of specific APIs in (Gatsby Node APIs)[/docs/node-apis/] for details.
```

Comment:
```suggestion
Some APIs provide additional helpers. For example `createPages` provides `graphql` function. Check documentation of specific APIs in [Gatsby Node APIs](/docs/node-apis/) for details.
```

fixup for markdown link syntax:
![Screenshot 2020-02-10 at 14 59 23](https://user-images.githubusercontent.com/419821/74156115-fc59f700-4c15-11ea-9b14-b15dabe16c9b.png)